aio/content/examples/template-syntax/src/app/hero.ts

```diff
@@ -11,7 +11,7 @@ export class Hero {
       325
     ),
     new Hero(1, 'Dr Nice',   'happy'),
-    new Hero(2, 'Narco',     'sad' ),
+    new Hero(2, 'RubberMan', 'sad' ),
     new Hero(3, 'Windstorm', 'confused' ),
```

Comment:
This is a pre-existing issue, but we can do a quick cleanup here as well (to remove an extra space):
```suggestion
    new Hero(2, 'RubberMan', 'sad'),
    new Hero(3, 'Windstorm', 'confused'),
```

Reply:
Thanks! Updated in latest commit.